ApiReview/API.Polly.Core/NoDocs/Polly.Retry/RetryStrategyOptions.cs

```diff
@@ -0,0 +1,25 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Polly.Strategy;
+
+namespace Polly.Retry;
+
+public class RetryStrategyOptions : ResilienceStrategyOptions
+{
+    public const int InfiniteRetryCount = -1;
+    public sealed override string StrategyType { get; }
+    [Range(-1, 100)]
+    public int RetryCount { get; set; }
+    public RetryBackoffType BackoffType { get; set; }
+    [TimeSpan("00:00:00", "1.00:00:00")]
+    public TimeSpan BaseDelay { get; set; }
+    [Required]
+    public Func<Outcome, ShouldRetryArguments, ValueTask<bool>>? ShouldRetry { get; set; }
```

Comment:
Can this have a default implementaiton that just returns true? If so, then you can make the property non-nullable.